decimal_test.go

```diff
@@ -743,3 +744,100 @@ func TestSizeof(t *testing.T) {
 		t.Errorf("sizeof(Context) changed: %d", s)
 	}
 }
+
+func TestUnmarshalText(t *testing.T) {
```

Comment:
math/big doesn't even test this stuff. Let's just remove this function and the one below. It's just a test of SetString at this point, which is tested elsewhere.

Reply:
`TestMarshalText()` tests `MarshalText()` with a `nil` `*Decimal`. Do you want me to keep `TestMarshalText()`, add this case as it's own test, or drop it completely?

Reply:
Remove TestUnmarshalText and TestMarshalText completely. This matches math/big.Int, and I'm ok with that.